internal/blocktx/store/postgresql/postgres_test.go

```diff
@@ -309,36 +373,149 @@ func TestPostgresDB(t *testing.T) {
 		stale4, err := postgresDB.GetBlock(ctx, hash4Stale)
 		require.NoError(t, err)
 		require.Equal(t, blocktx_api.Status_LONGEST, stale4.Status)
+
+		// when
+		err = postgresDB.UpdateBlocksStatuses(ctx, blockStatusUpdatesViolating)
+		require.True(t, errors.Is(err, store.ErrFailedToUpdateBlockStatuses))
```

Comment:
nit: You could use here `require.ErrorIs(t, actualErr, tc.expectedError)`

Reply:
Right, I fixed it in other places as well